include/uapi/sound/sof/abi.h

```diff
@@ -26,7 +26,7 @@
 
 /* SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 13
+#define SOF_ABI_MINOR 14
```

Comment:
There's some confusion now with ABI versions https://github.com/thesofproject/linux/pull/1929/
FYI @xiulipan

Reply:
I think I'll change this patch to 15.. to align with firmware PR.

Reply:
@singalsu Is that the consensus on what it should be? I noticed Liam put it in the ABI change classifier.

Reply:
No, obviously it can't be 14. I'll change this PR again when it's known @lgirdwood @plbossart .

Reply:
At 15 now but in above comment by @lyakh there's already PR with proposal for same so maybe this needs to be 16 for both kernel and FW. Waiting for guidance!

Reply:
it's 14 as per https://github.com/orgs/thesofproject/projects/2